content/docs/overview/tracers.md

```diff
@@ -1,3 +1,51 @@
 ---
 title: Tracers
 ---
+
+## Introduction
+
+OpenTracing provides an open, vendor-neutral standard API for consistent distributed tracing of request across processes, from web and mobile client platforms to the storage and custom backends at the bottom of application. This means that instrumentation would remain the same irrespective of the tracer system being used by the developer. In order to instrument the web application using OpenTracing specification, a compatible OpenTracing tracer must be deployed and listening for incoming span requests. A list of the all the supported tracers is available [here](./../../supported-tracers).
```

Comment:
> API for consistent distributed tracing of request across processes

A few thoughts:
  1. May be worth phrasing this more generically. First and foremost, OT provides an API for describing distributed transactions, specifically causality, semantics, and timing. What such description is used for is a secondary concern: distributed tracing is one usage, but not the only one.
  1. OT provides a general purpose distributed context propagation framework. It consists of API primitives for:
    * passing the metadata context in-process
    * encoding and decoding the metadata context for transmitting it over the network for inter-process communications
    * causality tracking: parent-child, forks, joins

I highly recommend [this paper][1] as a theoretical background and context of where OT fits in the distributed tracing space.

[1]: http://www.pdl.cmu.edu/PDL-FTP/SelfStar/CMU-PDL-14-102.pdf

Reply:
you're welcome. IMO that paper is a must-read for anyone interested in distributed tracing, it gives a great overview of the domain.